content/en/profiler/enabling/ssi.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Single Step Instrumentation for Profiling
+private: true
+---
+
+# Overview
+
+[Continuous Profiler][3] can be enabled as part of the [Single Step APM Instrumentation][1]
+(SSI) setup. Since this mode of profiler enablement is in private beta it is not yet integrated in
+the UI of the agent installation page. This page thus provides instructions on how to set it up. You
+should first familiarize yourself with SSI in general.
```

Comment:
The link to the SSI page is provided above on line 8. I do agree that this could be made more clear. Maybe moving the appeal to familiarization to the start of the paragraph and rephrasing it as:
"This document presumes you are already familiar with [Single Step APM Instrumentation][1] (SSI.) [Continuous Profiler][3] can be enabled as part of the SSI  setup."
WDYT?

Reply:
Okay, went ahead and made the change – called out the need to be familiar with SSI first, placed the link there, and then just referred to it as SSI later.